QUARKUS-2483.md

```diff
@@ -0,0 +1,75 @@
+# QUARKUS-2483 - Kubernetes service binding support for Reactive SQL clients
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2483
+
+PR: https://github.com/quarkusio/quarkus/pull/25657
+
+In Quarkus 2.10 was added Kubernetes service binding support for following Reactive SQL Clients extensions:
```

Comment:
Pls also mention product support level, you can consult https://code.quarkus.redhat.com, DB2 TP is ditched with 2.13 - https://issues.redhat.com/browse/QUARKUS-2332

Basically just PostgreSQL and MySQL ones are supported, MSSQL as TP, the rest just on community level.

Reply:
Done, thanks.